content/tutorials/themes/creating-themes/create-css/index.md

```diff
@@ -19,15 +19,23 @@ A CSS file can be associated with a layout template or container by giving it th
+3.  module.css ( ~/DesktopModules/mymodulename/module.css) — The CSS for every module type that appears on the page (ModuleCss).
+4.  resources.css ( ~/Resources/.../resourcename.css) — The CSS for resources such as bootstrap, cookie consent and search (ResourceCss).
+5.  skin.css ( ~/Portals/PortalID/Skins/SkinPackageName/skin.css) — The master CSS for the theme must be called skin.css (SkinCss).
+6.  MyThemeTemplate.css ( ~/Portals/_default/Skins/SkinPackageName/MyThemeTemplate.css) — The CSS for a specific layout template  (SpecificSkinCss).
+7.  container.css ( ~/Portals/PortalID/Containers/ContainerPackageName/container.css) — The master CSS for all containers in your theme (ContainerCss).
+8.  portal.css ( ~/Portals/PortalID/portal.css) — The CSS that can override elements in the installed themes. Website administrators can use this to override any styles in the theme or containers (PortalCss).
+9.  CustomPage.css ( ~/Portals/PortalID/MyCustomPage.css) —  File must be uploaded to portal root directory or sub-directory.  CustomPage.css files generally can share the same name as the page but is not required (example: Home = home.css). Set the file name in Persona Bar > Content > Pages select the page and navigate to Advanced > Appearance and set the CustomPage.css file name in the Page Stylesheet text area.  Currently loads in order of files included with DefaultPriority css files.
+10.  DefaultPriority.css ( Any location such as an external Url or a DNN site folder ) —  CSS files that get loaded with no priority in the DNN framework or a module which are applied in the order loaded.  CustomPage.css uses this priority and is generally first however this is determined by first come first serve basis and can change on different pages depending on which modules, skins and other resources are loading css files using this priority.  ("DefaultPriority" or no priority assigned).
 > [!NOTE]
 > These style sheets are not required. You can also store all your styles in one master theme style sheet (skin.css). However, if you combine your container styles and your theme styles in one CSS file, then the container will display properly only when used with your theme.
+>
+>  There are a few other css load order priorities not mentioned above.  SpecificContainerCss will load after ContainerCss. FeatureCss priority file feature.css and then IeCss priority file ie.css will load after AdminCss and prior to ModuleCss.  However these files are not currently being loaded by the platform.
+>
+>  Developers can use a css file load order priority determined by setting a CssPriority value when calling the method ClientResourceManager.RegisterStyleSheet(this, customStylesheet, CssPriority); to load a css file in desired order. More on this css file order can be found in the DNN Platform github source file currently located here: https://github.com/dnnsoftware/Dnn.Platform/blob/develop/DNN%20Platform/DotNetNuke.Web.Client/FileOrder.cs#L92
```

Comment:
```suggestion
>
> Theme and module developers can load their own CSS files by calling the method `ClientResourceManager.RegisterStyleSheet` or using the `DnnCssInclude` control.  If priority is not specified in these calls, `DefaultPriority` will be used.  For modules and themes, a priority at or near `ModuleCss` and `SkinCss` are probably more correct, so that themes can more easily override module styles and administrators can more easily override theme styles.
```